src/core/service.c

```diff
@@ -2758,6 +2761,59 @@ static void service_run_next_main(Service *s) {
         (void) service_set_main_pidref(s, TAKE_PIDREF(pidref), &s->main_command->exec_status.start_timestamp);
 }
 
+static bool service_should_reload_extensions(Unit *u) {
+        Service *s = ASSERT_PTR(SERVICE(u));
+        int r;
+
+        /* Only support this for notify-reload service types. */
```

Comment:
why, btw?

Reply:
see comment below

Reply:
please drop this. it just makes this harder to use for people

Reply:
this is counterproductive. please drop this.